.github/workflows/fpb-lint.yml

```diff
@@ -14,7 +19,26 @@ jobs:
       with:
         node-version: '16.x'
     - run: npm install -g free-programming-books-lint
-    - run: fpb-lint ./books/
-    - run: fpb-lint ./casts/
-    - run: fpb-lint ./courses/
-    - run: fpb-lint ./more/
+    - run: |
+        fpb-lint ./books/ 2>> error-log.txt | echo error-log.txt
```

Comment:
Has you tried `tee --append` command like Seth does in his workflow?  

https://github.com/SethFalco/free-programming-books/blob/e9f7dff3be87e640a72f97bd4eec1be79ec2b43a/.github/workflows/fpb-lint.yml#L17

I think it's more apropiate to print and conserve content of stdout stream

Reply:
using `tee` github didn't understand the errors

Reply:
`tee` output: /home/runner/work/free-programming-books/free-programming-books/books/free-programming-books-langs.md